message_definitions/v1.0/common.xml

```diff
@@ -211,6 +211,9 @@
       <entry value="536870912" name="MAV_SYS_STATUS_OBSTACLE_AVOIDANCE">
         <description>0x20000000 Avoidance/collision prevention</description>
       </entry>
+      <entry value="1073741824" name="MAV_SYS_STATUS_SENSOR_PORPULSION">
+        <description>0x40000000 porpulsion (actuator, esc, motor or propellor)</description>
```

Comment:
Spelling needs fixing.

That would be `PROPULSION`.

Reply:
done